cmake/config.cmake

```diff
@@ -140,7 +140,7 @@ set(USE_MKLDNN OFF)
 set(USE_OPENMP none)
 
 # Whether use contrib.random in runtime
-set(USE_RANDOM OFF)
+set(USE_RANDOM ON)
```

Comment:
Would this flag name too vague? At least we should improve the description by mentioning what happen if we set it to OFF.

Reply:
IMO, I think it is ok to keep this flag name, because it is indeed `random` functionality related stuffes. I think it is similiar as the `graph runtime` / `sort` which is always been turned on and shouldn't been turned off if there is no special reason.